scripts/download_data.py

```diff
@@ -0,0 +1,13 @@
+import bokeh.sampledata
```

Comment:
Changed from bash to python because there was no need for it to be a bash file...